### PR TITLE
Added better code rendering but failed at pagination for profile view (will work in new PR)

### PR DESCRIPTION
**Pull request by davodesign84**

### CODE REVIEW

File: packages/kauri-web/ant-override-vars.less [hunk 1]
```diff
@@ -25,3 +26,14 @@
 @link-decoration: #0ba986;
 
 @item-hover-bg: #0ba986;
+
+.DescriptionRow-markdown pre, .mde-preview-content pre {
```
Comment (rej156) [inline packages/kauri-web/ant-override-vars.less]: Should we move this to the other .less file?

File: packages/kauri-web/components/containers/PublicProfile/index.js [hunk 2]
```diff
@@ -18,24 +18,30 @@ export default compose(
   graphql(searchPersonalArticles, {
     name: 'ArticlesQuery',
     options: ({userId}) => ({
+      fetchPolicy: 'cache-and-networ',
```
Comment (rej156) [inline packages/kauri-web/components/containers/PublicProfile/index.js]: cmd+d
Correct typo

File: packages/kauri-web/queries/Article.js [hunk 3]
```diff
@@ -180,7 +180,7 @@ export const searchApprovedArticles = gql`
 `
 
 export const globalSearchApprovedArticles = gql`
-  query globalSearchApprovedArticles($size: Int = 12, $page: Int = 0, $text: String) {
+  query globalSearchApprovedArticles($size: Int = 666, $page: Int = 0, $text: String) {
```
Comment (rej156) [inline packages/kauri-web/queries/Article.js]: Evil

File: packages/kauri-web/components/containers/PublicProfile/index.js (hunk 2 above)
Comment (davodesign84) [reply packages/kauri-web/components/containers/PublicProfile/index.js]: Good spot!

File: packages/kauri-web/queries/Article.js (hunk 3 above)
Comment (davodesign84) [reply packages/kauri-web/queries/Article.js]: ahahh Easter Egg :D

File: packages/kauri-web/ant-override-vars.less (hunk 1 above)
Comment (davodesign84) [reply packages/kauri-web/ant-override-vars.less]: moved it to `custom-css.less`

Comment (rej156) [reply packages/kauri-web/ant-override-vars.less]: Meant the markdown-rendered-css.less file :P

**Pull request merged**